pom.xml

```diff
@@ -544,7 +545,7 @@
           <dependency>
             <groupId>org.codehaus.mojo</groupId>
             <artifactId>extra-enforcer-rules</artifactId>
-            <version>1.0-beta-4</version>
+            <version>1.0-beta-7-SNAPSHOT</version>
```

Comment:
I need to know how to deploy this snapshot /cc @stephenc

Reply:
@recena anyone who is a maintainer of any plugin can release a SNAPSHOT of any other. Just do `mvn clean deploy`.

(IIUC/IIRC, it's just that the handle must appear somewhere in https://github.com/jenkins-infra/repository-permissions-updater/ permissions, amirite @daniel-beck?)

Reply:
Oh wait, this is extra-enforcer-rule, hence MojoHaus, not Jenkins org. So I can do it too if really needed. We don't have the same policy there, though.

Reply:
@batmat I would appreciate it.